src/aat/resources/features/F-029/S-051.td.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "must return appropriate negative response when request does not provide an authorized access",
```

Comment:
better wording please - it's not a request to provide an authorised or unauthorised access.

Reply:
updated.